web/src/utils/submitListUtils.ts

```diff
@@ -107,8 +109,9 @@ const constructRegistrationTemplate = (listMetadata: IListMetadata) => {
     category: "Curated Lists",
 
     policyURI: listMetadata.policyURI,
-    frontendUrl: window.origin,
-    arbitratorChainID: "421614",
+    frontendUrl: `${netlifyUri}/#/lists/item/{{ itemID }}@{{listAddress}}`,
+    //TODO : this will depend on the chain the dispute is created on?
```

Comment:
For production we should do something like this: https://github.com/kleros/kleros-v2/blob/dev/web/src/consts/chains.ts#L4